.github/workflows/go.yml

```
@@ -49,3 +44,8 @@ jobs:
         file sandcat.go-linux && ls -al sandcat.go-linux && ./sandcat.go-linux -help
       env:
         GOOS: linux
+
+    - name: Test Shells Executors
+      run: |
+        cd gocat
+        go test -v ./execute/shells/...
```

```suggestion
    - name: Test Sandcat Agent
      run: |
        cd gocat
        go test -v ./...
```

Do you think we could change this into this generic form, this way prevent changing the workflow too much and new tests would be discovered automatically. Tested locally, but it would be good to confirm with the workflow.